### DISCUSSION TITLE
0.83.0

### DISCUSSION THREAD

**Discussion by achantavy** [cartography-cncf/cartography#1229] [General]
## What's Changed
* Add contributing guidelines for issues by @achantavy in https://github.com/lyft/cartography/pull/1226
* #1216: AWS SSM->new data model by @achantavy in https://github.com/lyft/cartography/pull/1217
* Adds support to query Semgrep API to ingest SCA vulns by @heryxpc in https://github.com/lyft/cartography/pull/1224
* 0.83.0 by @ramonpetgrave64 in https://github.com/lyft/cartography/pull/1228

**Full Changelog**: https://github.com/lyft/cartography/compare/0.82.0...0.83.0

<hr /><em>This discussion was created from the release <a href='https://github.com/lyft/cartography/releases/tag/0.83.0'>0.83.0</a>.</em>

**Comment by jychp** [comment]
Information are available in [release note](https://github.com/cartography-cncf/cartography/releases/tag/0.83.0).
